CustomScript/customscript.py

```diff
@@ -322,7 +322,13 @@ def download_blob(storage_account_name, storage_account_key,
                                         blob_uri,
                                         download_dir)
         blob_name, _, _, download_path = result
-        preprocess_files(download_path, hutil)
+        enable_dos_converting = True
+        public_settings = hutil.get_public_settings()
+        if public_settings:
+            if 'enableDOSConverting' in public_settings:
```

Comment:
is `enableDOSConverting` being newly introduced? if so we need a docs update as well